lib/active_record/connection_adapters/trilogy_adapter.rb

```diff
@@ -30,11 +30,11 @@ def explain(arel, binds = [])
           MySQL::ExplainPrettyPrinter.new.pp(result, elapsed)
         end
 
-        def execute(sql, name = nil, async: false)
+        def execute(sql, name = nil, async: false, allow_retry: false)
```

Comment:
It looks to me like this method is identical to the one in activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb that it inherits from. If I'm correct about that, can rm this definition and rely on the inherited one instead?

Reply:
Yeah, that would make a lot more sense eh? 😂 I've removed the definition as well as the test (since relying on the upstream tests seems sufficient). I also dropped the entry from the changelog since it felt unnecessary at this point, but let me know if I should put that back.

Reply:
That all makes sense to me. Thanks again!

I'm not sure why CI isn't running against this PR, so I'll run the tests locally and then merge.